src/pages/iou/WaypointEditor.js

```diff
@@ -263,7 +265,7 @@ export default withOnyx({
         // Only grab the most recent 5 waypoints because that's all that is shown in the UI. This also puts them into the format of data
         // that the google autocomplete component expects for it's "predefined places" feature.
         selector: (waypoints) =>
-            _.map(waypoints ? waypoints.slice(0, 5) : [], (waypoint) => ({
+            _.map(waypoints ? waypoints.slice(0, 20) : [], (waypoint) => ({
```

Comment:
The app still shows only 5 predefined places, looks like this line in [IOURequestStepWaypoint.js](https://github.com/Expensify/App/blob/2f02fdd71ba5ec354651c7dcb2019f2f4c2d25f0/src/pages/iou/request/step/IOURequestStepWaypoint.js#L266) needs to be changed. I think we can even add a variable to CONST.js and use it in all 3 files